modules/partition-replicator/src/test/java/org/apache/ignite/internal/partition/replicator/ZoneResourcesManagerTest.java

```diff
@@ -59,6 +60,9 @@ class ZoneResourcesManagerTest extends IgniteAbstractTest {
 
     private ZoneResourcesManager manager;
 
+    // TODO https://issues.apache.org/jira/browse/IGNITE-24654 Ensure that tracker is closed.
+    private PendingComparableValuesTracker<Long, Void> storageIndexTracker;
```

Comment:
Why don't you close it in this test?

Reply:
I believe that given test should verify that it's closed, whether storageIndexTracker close in not triggered yet. It'll be covered in https://issues.apache.org/jira/browse/IGNITE-24654